alert-common/src/main/java/com/synopsys/integration/alert/common/annotation/SensitiveFieldFinder.java

```diff
@@ -35,16 +32,6 @@ private SensitiveFieldFinder() {
         throw new IllegalStateException("Utility class");
     }
 
-    public static Set<Field> findSensitiveFields(final Class<?> clazz) {
-        final Set<Field> fields = new HashSet<>();
-        for (final Field field : clazz.getDeclaredFields()) {
-            if (field.isAnnotationPresent(SensitiveField.class) || hasParentSensitiveAnnotation(field.getAnnotations())) {
-                fields.add(field);
-            }
-        }
-        return fields;
-    }
-
     public static boolean hasParentSensitiveAnnotation(final Annotation[] annotations) {
```

Comment:
If this method is still used, does it still make sense to be in this class? or do we have other classes that deal with fields that this could move into?

Reply:
IMO this class should be removed. It is used on a single field to prevent its toString from printing a secure value. We should instead modify that specific objects toString to ignore that field and then remove all classes related to this sensitive field annotation.